pkg/api/v20210131preview/openshiftcluster.go

```diff
@@ -116,13 +116,22 @@ type ServicePrincipalProfile struct {
 
 // NetworkProfile represents a network profile.
 type NetworkProfile struct {
+	// The Network Provider to use when installing the cluster.
+	NetworkType string `json:"networkType,omitempty"`
+
 	// The CIDR used for OpenShift/Kubernetes Pods.
 	PodCIDR string `json:"podCidr,omitempty"`
 
 	// The CIDR used for OpenShift/Kubernetes Services.
 	ServiceCIDR string `json:"serviceCidr,omitempty"`
 }
 
+// NetworkType constants.
+const (
```

Comment:
This needs to be Custom Type (as per 1200)

Reply:
Sure, I will make this change shortly

Reply:
Changed as requested.